tests/fixtures/ecma-version/6/modules/export-from-named-as-default.result.js

```diff
@@ -183,7 +183,7 @@ export default {
             ]
         },
         {
-            "type": "Keyword",
+            "type": "Identifier",
             "value": "default",
```

Comment:
Is this another bug?

Reply:
I think so, `default` is a reversed word in js. /cc @marijnh

Reply:
filed in acorn: https://github.com/acornjs/acorn/issues/1273

Reply:
Thanks for following up on this. I'm not sure if we should treat this as a bug fix or a breaking change, or if we should modify Espree to output the "Keyword" as it did before. @mdjermanovic what do you think?

Reply:
This is `default` in `export {foo as default} from "foo";`?

In this context, `default` doesn't seem to have any particular syntactical meaning and is thus just a name like any other, so I think that `"type": "Identifier"` makes more sense than ``"type": "Keyword"``. Also, in the specification for [Exports](https://tc39.es/ecma262/#sec-exports), it doesn't seem that `default` in this context appears as a terminal symbol (bolded) in any productions, which I think is a rule of thumb for whether something is a keyword or not.

I think we can classify this as a bug fix.

Reply:
Sounds good. :1: